defaults.properties.yml

```diff
@@ -248,3 +249,11 @@ phpmd:
 #     http://docs.behat.org/en/v2.5/guides/7.config.html#profiles)
 behat:
   args: "--suite=default --strict"
+
+
+# Configuration for composer.
+#
+# Use this configuration if you're running on an environment where composer is not available on the command line as
+# `composer` or `composer.phar`.
+# composer:
+#   composer: ${build.thebuild.dir}/vendor/bin/composer.phar
```

Comment:
Would this require us to have `"composer/composer": "^1.7"` (or something similar) as a dependency for the-build (like https://github.com/palantirnet/the-build/pull/34/files)?  Also, I did some local testing of this method by moving `composer` out of my path in the VM, and I had to use `vendor/bin/composer` as the path in this config.

Reply:
I tweaked this description.

Reply:
Basically, you may or may not want to add composer to your project... if you're checking dependencies in to your repo, then sure, but otherwise you may just have it in a non-standard location.

Reply:
That's helpful!  Thanks for clearing that up.